sunpy/net/dataretriever/sources/stereo.py

```diff
@@ -0,0 +1,366 @@
+"""
+This module implements STEREO Client.
+"""
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+
+__author__ = "Sudarshan Konge"
+__email__ = "sudk1896@gmail.com"
+
+import datetime
+import re
+
+from sunpy.net.dataretriever.client import GenericClient
+from sunpy.util.scraper import Scraper
+
+__all__ = ['SECCHIClient', 'PLASTICClient', 'IMPACTClient', 'SWAVESClient']
+
+class SECCHIClient(GenericClient):
+    """
+    Returns a list of URLS to SECCHI files corresponding to value of input timerange.
+    URL source: `http://stereo-ssc.nascom.nasa.gov/data/beacon/`.
+    Parameters
+    ----------
+    timerange: sunpy.time.TimeRange
+        time range for which data is to be downloaded.
+        Example value - TimeRange('2015-12-30 00:00:00','2015-12-31 00:01:00')
+
+    Instrument: Fixed argument = 'secchi'
+
+    Source: Two arguments, case-sensitive - 'STEREO_A' or 'STEREO_B'.
```

Comment:
@wafels: I have stuck to VSO conventions. That is how they call it and hence I haven't. And @dpshelio told me to use these values as well (It is somewhere here in the comments, I believe).